install-dev/data/xml/tab.xml

```diff
@@ -546,7 +546,7 @@
             <wording>Webservice</wording>
             <wording_domain>Admin.Navigation.Menu</wording_domain>
         </tab>
-        <tab id="AuthorizationServer" id_parent="Advanced_Parameters" active="0" enabled="1">
+        <tab id="AuthorizationServer" id_parent="Advanced_Parameters" active="1" enabled="1">
```

Comment:
I also had to change the default value of the tab, otherwise it is not displayed, even if the feature flag is true